orchestration/dags/import_analytics.py

```diff
@@ -286,6 +286,26 @@ def getting_service_account_token():
     dag=dag,
 )
 
+create_offer_extracted_data = BigQueryOperator(
+    task_id="create_offer_extracted_data",
+    sql=f"""Select B.offer_id, B.offer_type, json_extract(offer_extra_data, "$.author") AS Author, 
+            json_extract(offer_extra_data, "$.performer") AS Performer, 
```

Comment:
C'est mieux de mettre les fonction BigQuery en majuscules
```suggestion
            JSON_EXTRACT(offer_extra_data, "$.performer") AS Performer, 
```